stdlib/hash.gr

```diff
@@ -1,16 +1,19 @@
 /**
  * @module Hash: Utilities for hashing any value.
- * @example import Hash from "hash"
+ * @example include "hash"
  *
  * @since v0.1.0
  */
 
+module Hash
```

Comment:
Does the formatter put this space between?

Reply:
Yes it does.

Reply:
Can we not do that?

Reply:
I suppose it's a stylistic thing, but I much prefer
```
module Main

print("Hello world")
```

to 

```
module Main
print("Hello world")
```

Plus separating that module statement from a block of include statements, etc.

Reply:
Sorry for not being clear. I was talking about the newline *above* the `module` statement (between `module Hash` and the module docblock comment).

Reply:
Oh, yes it does, though that wasn't really my intention. The formatter seems to consider regular blocks and docblocks differently, where if a line wasn't forced here, if you started your file with grainc-flags, for example, it would print the flags like this:
```
/* grainc-flags --no-gc */ module Main
```

We'll probably need to completely rework how start-of-module comments are printed.

Reply:
Please open an issue because this really needs to be fixed. Graindoc has a hack that checks 2 lines backward (due to attributes) which would be the only way this is working (though I don't think Graindoc is even updated to match the module to the docblock)

Reply:
https://github.com/grain-lang/grain/issues/1635